hw/ip/otp_ctrl/data/otp_ctrl_sec_cm_testplan.hjson

```diff
@@ -27,271 +27,271 @@
+      tests: ["otp_ctrl_parallel_lc_esc", "otp_ctrl_sec_cm"]
     }
     {
       name: sec_cm_part_data_reg_integrity
       desc: "Verify the countermeasure(s) PART.DATA_REG.INTEGRITY."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_init_fail"]
     }
     {
       name: sec_cm_part_data_reg_bkgn_chk
       desc: "Verify the countermeasure(s) PART.DATA_REG.BKGN_CHK."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_check_fail"]
     }
     {
       name: sec_cm_part_mem_regren
       desc: "Verify the countermeasure(s) PART.MEM.REGREN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_part_mem_sw_unreadable
       desc: "Verify the countermeasure(s) PART.MEM.SW_UNREADABLE."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_part_mem_sw_unwritable
       desc: "Verify the countermeasure(s) PART.MEM.SW_UNWRITABLE."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_lc_part_mem_sw_noaccess
       desc: "Verify the countermeasure(s) LC_PART.MEM.SW_NOACCESS."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_access_ctrl_mubi
       desc: "Verify the countermeasure(s) ACCESS.CTRL.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_token_valid_ctrl_mubi
       desc: "Verify the countermeasure(s) TOKEN_VALID.CTRL.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_lc_ctrl_intersig_mubi
       desc: "Verify the countermeasure(s) LC_CTRL.INTERSIG.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_test_bus_lc_gated
       desc: "Verify the countermeasure(s) TEST.BUS.LC_GATED."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_direct_access_config_regwen
       desc: "Verify the countermeasure(s) DIRECT_ACCESS.CONFIG.REGWEN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_regwen"]
     }
     {
       name: sec_cm_check_trigger_config_regwen
       desc: "Verify the countermeasure(s) CHECK_TRIGGER.CONFIG.REGWEN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_check_config_regwen
       desc: "Verify the countermeasure(s) CHECK.CONFIG.REGWEN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_macro_mem_integrity
       desc: "Verify the countermeasure(s) MACRO.MEM.INTEGRITY."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_macro_err"]
     }
     {
       name: sec_cm_macro_mem_cm
       desc: "Verify the countermeasure(s) MACRO.MEM.CM."
       milestone: V2S
-      tests: []
+      tests: [N/A]
```

Comment:
From hjson description, this might vary depends on each vendor.
https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L1063
I could put a generic `otp_ctrl_macro_err` test, but it really depends on how vendor's DV engineer plan to verify it.